collections/README.md

```diff
@@ -879,8 +879,8 @@ assertEquals(withoutList, [3]);
 
 ### zip
 
-Builds 2-tuples of elements from the given array with matching indices, stopping
-when the smaller array's end is reached
+Builds N-tuples of elements from the given N arrays with matching indices,
+stopping when the smaller array's end is reached
```

Comment:
Should maybe be

```suggestion
stopping when the smallest array's end is reached
```

now

Reply:
I forgot to do that in https://github.com/denoland/deno_std/pull/1204/commits/0fefab0b2e0b6da4eb2677239136e9684ef935d2. Thanks for catching